<launch>

    <node pkg="usb_cam" type="usb_cam_node" name="cam1">
	  	<param name="video_device" value="/dev/video0" />
        <param name="framerate" value="15" />	
        <remap from="/cam1" to="/cam1/camera"/>
	</node> 

	<node name="tag_detector" pkg="competition_4" type="tag_detector.py" output="screen"/>

</launch>
